configure.py

```diff
@@ -856,6 +856,7 @@ def find_ninja():
                 'sstables/sstables_manager.cc',
                 'sstables/sstable_set.cc',
                 'sstables/storage.cc',
+                'sstables/mirrored_storage.cc',
```

Comment:
A note for future: if putting this new storage into own .cc file, then filesystem_ and s3_ storages deserve the same